src/com/javarush/task/task08/task0829/Solution.java

```diff
@@ -0,0 +1,101 @@
+package com.javarush.task.task08.task0829;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+Модернизация ПО
+*/
+
+public class Solution {
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        HashMap<String, String> list = new HashMap<>();
```

Comment:
Не обманывай, это не list, а map. Во вторых используй абстракицю:
```suggestion
        Map<String, String> map= new HashMap<>();
```